janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticSearchConfigTest.java

```diff
@@ -241,6 +241,37 @@ public void testExternalDynamic() throws Exception {
         idx.close();
     }
 
+    @Test
+    public void testExternalDynamicWithDynamicTemplate() throws Exception {
+        final Duration maxWrite = Duration.ofMillis(2000L);
+        final String storeName = "test_mapping";
+        final Configuration indexConfig = GraphDatabaseConfiguration.buildGraphConfiguration().set(USE_EXTERNAL_MAPPINGS, true, INDEX_NAME).set(USE_DYNAMIC_TEMPLATE, true, INDEX_NAME).restrictTo(INDEX_NAME);
+        final IndexProvider idx = open(indexConfig);
+        final ElasticMajorVersion version = ((ElasticSearchIndex) idx).getVersion();
+
+        //Test create index KO mapping is not push
+        final KeyInformation.IndexRetriever indexRetriever = IndexProviderTest
+            .getIndexRetriever(IndexProviderTest.getMapping(idx.getFeatures(), ANALYZER_ENGLISH, ANALYZER_KEYWORD));
+        final BaseTransactionConfig txConfig = StandardBaseTransactionConfig.of(TimestampProviders.MILLI);
+        final IndexTransaction itx = new IndexTransaction(idx, indexRetriever, txConfig, maxWrite);
+        try {
+            idx.register(storeName, "date", IndexProviderTest.getMapping(idx.getFeatures(), ANALYZER_ENGLISH, ANALYZER_KEYWORD).get("date"), itx);
+            fail("should fail");
+        } catch (final PermanentBackendException e) {
```

Comment:
Not even an error or log message? This is silently catching errors and hiding them from users, which may make debugging hard and frustrating in the future.

Reply:
It was a copy/past from an other test.
So I add log on all tests of this file.